CONTRIBUTING.md

```diff
@@ -46,7 +46,7 @@ facilitates social contribution, easy testing and peer review.
 
 To contribute a patch, the workflow is as follows:
 
-  1. Fork repository
+  1. Fork repository ([only the first time](https://help.github.com/en/articles/configuring-a-remote-for-a-fork)).
```

Comment:
Maybe link to https://help.github.com/en/articles/fork-a-repo is more relevant here?

Reply:
Yes this is more relevant to the topic. I'll change this

Reply:
What about one level closer to the source?
https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project

Reply:
Bit still, I don't get it. They mention `upstream` also in the Pro Git book.

What is wrong about this?

```
git clone https://github.com/bitcoin/bitcoin
cd bitcoin
git remote add myfork git@github.com:...
```

I.e. `origin` is `upstream` here and there is no need to write the whole word of `upstream` or change the `master` branch to be pulled from `upstream` remote...

Reply:
Above is a request for comments. Thanks.